_posts/2020-12-24-how-to-start-with-foreman

```diff
@@ -0,0 +1,399 @@
+---
+layout: post
+title: How to start with Foreman
+date: 2020-12-24 17:00:00
+author: Marek Hulan
+tags:
+- foreman
+- community
+- tutorial
+- howto
+- beginners
+---
+
+This post targets primarily users new to Foreman. It can be interesting for those,
+who would like to start using Foreman but feel lost or who just installed Foreman
+and would like to investigate its capabilities. We'll assume a fresh Foreman
+installation without any plugin and with the default configuration. The primary focus 
+will be on on filling in the data about the infrastructure. If you're searching for
+installation instructions, please take a look at [this chapter](https://theforeman.org/manuals/2.3/index.html#3.InstallingForeman)
+of the Foreman manual. This post assumes installation using the foreman-installer.
+
+<!--more-->
+
+Foreman allows to define many objects, which serve different purposes. But the
+heart of everything is the Host object, It glues all other objects together.
+Host represents some machine. It can be a device on network or simply something,
+that you want to install operating system. It's typically a thing you want to manage
+or automate some tasks around it.
```

Comment:
```suggestion
A host represents a machine. It can be a device on the network or simply something,
that you want to install an operating system on. It's typically a thing you want to manage
or automate some tasks around it.
```